test/locale/fallbacks_test.rb

```diff
@@ -130,4 +130,12 @@ def setup
     @fallbacks.map(:no => :nb, :nb => :no)
     assert_equal [:nb, :no, :"en-US", :en], @fallbacks[:nb]
   end
+
+  # Test I18n::Disabled  is raised correctly when locale is false during fallback
+
+  test "with locale equals false" do
```

Comment:
Let me know if you actually prefer this one to be in its own test class. I wasn't sure about adding it to this one.

Reply:
Seems fine :)